src/azul/plugins/__init__.py

```diff
@@ -129,6 +132,13 @@ def service_config(self) -> ServiceConfig:
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def aggregate_class(self) -> Type['Aggregate']:
```

Comment:
```suggestion
    def aggregate_class(self) -> Type[Aggregate]:
```

https://www.python.org/dev/peps/pep-0484/#forward-references